### PR TITLE
{phys}[foss/2019b] GetFEM v5.4.1

### PR DESCRIPTION
**Pull request by jordiblasco**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+builddependencies = [
+    ('Autoconf', '2.69', '', ('GCCcore', '8.3.0')),
+    ('Automake', '1.16.1', '', ('GCCcore', '8.3.0')),
+    ('Autotools', '20180311', '', ('GCCcore', '8.3.0')),
+]
+
+dependencies = [
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb:29]: @jordiblasco I think you're missing the `Python` dependency here.

File: easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# #
+
+easyblock = 'ConfigureMake'
+
+name = 'GetFEM'
+version = '5.4.1'
+local_pyver = '3.7.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://getfem.org'
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb:15]: @jordiblasco do you know what's going on with the homepage? I tried the `https` version of that address, and it had a strange result.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb (hunk 2 above)
Comment (verdurin) [reply easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb:15]: @jordiblasco we still need to deal with the homepage error - can it be `https` or not?

File: easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+
+name = 'GetFEM'
+version = '5.4.1'
+local_pyver = '3.7.4'
+versionsuffix = '-Python-%(local_pyver)s'
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb:13]: Please remove the `local_pyver`, change `versionsuffix` back to normal, and add `Python` as a dependency.

File: easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+builddependencies = [
+    ('Autoconf', '2.69', '', ('GCCcore', '8.3.0')),
+    ('Automake', '1.16.1', '', ('GCCcore', '8.3.0')),
+    ('Autotools', '20180311', '', ('GCCcore', '8.3.0')),
+]
+
+dependencies = [
+    ('Qhull', '2019.1', '', ('GCCcore', '8.3.0')),
+    ('MUMPS', '5.2.1', '', ('foss', '2019b-metis')),
+    ('SciPy-bundle', '2019.10', '', ('foss', '2019b-Python-%(local_pyver)s')),
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GetFEM/GetFEM-5.4.1-foss-2019b-Python-3.7.4.eb:32]: With the changes listed above, please change this to be `versionsuffix`, in order to pick up the correct `SciPy-bundle` dependency.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Review by jordiblasco — COMMENTED** [review]
@verdurin from my point of view if the HTTPS homepage is not working as expected, it shouldn't be our problem. Use HTTP instead.

**Review by jfgrimm — COMMENTED** [review]
As decided in issue #16330, we have deprecated the use of `True` to signify a system-toolchain dependency (#16384), in favour of the more intuitive `SYSTEM` template constant. Due to the change in the test suite, please run `eb --sync-pr-with-develop 11685` and update the PR to use `SYSTEM` instead.

**Comment by akesandgren** [comment]
Closing this PR, since it (only) uses deprecated toolchain versions,
see https://docs.easybuild.io/policies/toolchains .

**Pull request closed without merging**